lib/services/elasticsearch.js

```diff
@@ -285,12 +294,24 @@ function ElasticSearch(kuzzle, options, config) {
    */
   this.create = function elasticsearchCreate (requestObject) {
     var
-      data = cleanData.call(this, requestObject);
+      data = cleanData(requestObject);
 
     if (data.body._routing) {
       Promise.reject(new BadRequestError('Kuzzle does not support "_routing" in create action.'));
     }
 
+    if (data.hasOwnProperty('refresh')) {
+      if (compareVersions(this.esVersion.number, '5.0.0') < 0) {
+        Promise.reject(new BadRequestError(`Refresh parameter is not supported by the version "${this.esVersion.number}" of Elasticsearch`));
+      }
+      if (compareVersions(config.apiVersion, '5.0') < 0) {
+        Promise.reject(new BadRequestError(`Refresh parameter is not supported by the version "${config.apiVersion}" of Elasticsearch API`));
+      }
+      if (data.refresh !== 'wait_for' && data.refresh !== 'false' && data.refresh !== false) {
+        Promise.reject(new BadRequestError('Refresh parameter only supports the value "wait_for" or false'));
```

Comment:
As of [elasticsearch 5.x documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html) we can also set the `refresh` to `true`